.travis.yml

```diff
@@ -69,6 +69,8 @@ notifications:
   email: false
 deploy:
   provider: puppetforge
+  deploy:
+    branch: ha-bug-puppet-forge
```

Comment:
This is based on https://github.com/travis-ci/dpl/issues/487
We currently can't deploy on ruby231 so we use a specific branch of the dpl gem that has a fix for that. (dpl is a travis internal gem that is installed when we want to publish to the forge).
https://github.com/voxpupuli/modulesync_config/commit/9d3419624ba62e54b58881c100776fa5b8ef2d8d